pythermalcomfort/models/thi.py

```
@@ -0,0 +1,49 @@
+    ----------
+    tdb : float or list of floats
+        Dry bulb air temperature, [°C].
+    rh: float or list of floats
+        Relative humidity, [%].
+    round_output : bool, optionalruff format
+        If True, rounds output value. If False, it does not round it. Defaults to True.
+
+    Returns
+    -------
+    THI
+        A dataclass containing the Temperature-Humidity Index. See :py:class:`~pythermalcomfort.classes_return.THI` for more details.
```

Pylint (3.3.7)

[convention] Line too long (133/100)

(C0301)